src/crimson/os/seastore/seastore_types.h

```diff
@@ -215,6 +217,16 @@ constexpr paddr_t make_fake_paddr(segment_off_t off) {
 constexpr paddr_t zero_paddr() {
   return paddr_t{ZERO_SEG_ID, 0};
 }
+constexpr paddr_t delayed_temp_paddr(segment_off_t off) {
+  return paddr_t{DELAYED_TEMP_SEG_ID, off};
+}
+
+enum class delay_inline_ool_t {
+  NO_DELAY,
+  DELAYED,
+  DELAYED_INLINE,
+  DELAYED_OOL
```

Comment:
From these names, it's not obvious what DELAYED means.  Every one of these names should have a one line description.

Reply:
See my comment on SegmentedAllocator::Writer::finish_write.